docs/api-testing/import-export-tests.md

```diff
@@ -1,36 +1,38 @@
 ---
 id: import-export-tests
 title: Importing and Exporting API Tests
-sidebar_label: Import and Export Tests
-description: Documentation for importing and exporting existing API tests.
+sidebar_label: Importing Tests
+description: How to import existing API tests.
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-Importing and exporting your existing API tests can be useful for:
-* Moving tests to another Project
+Importing and exporting existing tests from within Sauce Labs API Testing can be useful for:
+* Moving tests from one Project to another
 * Making another API Fortress deployment
```

Comment:
Hi @LoryApiFortress , what is this referring to?

Reply:
Hi @kimsauce this statement has been copied from the Legacy doc and it should referring to the on-prem version: you had to export the existing test to avoid to loose some test while deploying a new version of APIF (MK2). You can remove it.